solutions/loading-web-fonts/pages/_app.tsx

```diff
@@ -15,7 +37,9 @@ function App({ Component, pageProps }: AppProps) {
       path="solutions/loading-web-fonts"
       description="How to correctly load web fonts"
     >
-      <Component {...pageProps} />
+      <div className={inter.variable}>
```

Comment:
I don't love advertising the variable, but it definitely makes sense for this example!

Reply:
Thinking about it I think this could be improved!

Reply:
Improved with the last commit, the example is now using `className`.